src/net/sourceforge/kolmafia/VYKEACompanionData.java

```diff
@@ -132,22 +132,24 @@ public class VYKEACompanionData implements Comparable<VYKEACompanionData> {
     "level 5 lightning lamp",
   };
 
-  public static final int NONE = 0;
-
-  public static final int BOOKSHELF = 1;
-  public static final int DRESSER = 2;
-  public static final int CEILING_FAN = 3;
-  public static final int COUCH = 4;
-  public static final int LAMP = 5;
-  public static final int DISHRACK = 6;
+  public enum VYKEACompanionType {
+    NONE,
+
```

Comment:
Is there a reason for a blank line here?

Reply:
there was a blank line in the original file between NONE and the rest, so I kept it.